specs/0.1.json

```diff
@@ -12,19 +13,20 @@
 
         "address": {
             "type":"object",
+            "additionalProperties": false,
             "description": "The address of the place you meet",
             "properties": {
-                "name": {
+                "Name": {
```

Comment:
we don't want to change that in existing schema files, as all api files will get invalid

Reply:
I'm sorry, that I haven't updated them. When I used git rebase to merge the conflicts, those commits applied to my fork. The changes that you are seeing is your last commit and my commits.